good_bot/cli.py

```diff
@@ -0,0 +1,18 @@
+import argparse
+
+
+from good_bot import Args, server
```

Comment:
I suggest a single blank line between groups of imports, unless we have a good reason not to?

sauce:
> Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
https://peps.python.org/pep-0008/#imports

Reply:
Auto formatters will help here